src/voxel.py

```diff
@@ -105,6 +105,8 @@ def __init__(self):
     def _initialise_data(self):
         # Our scene data
         self._data = self.blank_data()
+        # Create empty selection
+        self._selection = ()
```

Comment:
this would create a tuple, which does not have a append or remove method, and not a [set](https://docs.python.org/2/tutorial/datastructures.html#sets)
```python
self._selection = set()
```